exercise/static/exercise/results_staff.js

```diff
@@ -1174,8 +1174,12 @@
              */
             function renderParticipantLink(data, type, row) {
                 // TODO: Get the link to students in a proper way
-                const link = $('li.menu-participants').find('a').attr('href');
-                return (row['UserID'] > 0 ? '<a href="' + link + row['UserID'] + '">' + data + '</a>' : '');
+                if(data == '—') {
+                    return '<span style="color: #005eb8;">' + data + '</span>';
```

Comment:
Is the inline style needed? How did the table look without this color change? Did you consider using a class attribute and a style sheet (separate file)? There is already a file `results_staff.css` that is used in this same page.

Reply:
I started to wonder why you actually removed the link when there is no student id. Even without a student id, the user still has a participant page for teachers and it makes sense to link there. Sure, the link is also available in the name column and the mdash is very "small" as a clickable link text. But is it a problem anyway?

Reply:
I added css because after removing link the color became black and rest of the text in table was blue. And there is no problem keeping the link on dash, I just thought dash would be too small to be a clickable link. I will add the link back and css will not be needed then.